gms/api/src/main/pegasus/com/linkedin/dashboard/Chart.pdl

```diff
@@ -35,4 +36,9 @@ record Chart includes ChartKey {
    * Status information for the chart such as removed or not
    */
   status: optional Status
+
+  /**
+   * List of global tags applied to the chart
+   */
+  globalTags: optional GlobalTags
```

Comment:
I don't think modeling "globalTags" as the property of every metadata entity is a good idea. Is there a way to decouple the tags from below entities?

- Chart
- Dashboard
- Dataset
- CorpUser

@jplaisted into discussion

Reply:
@woodyzhou just to clear- this is the external API definition of the entity, the format we expose to clients. The internal model of the entity accepts tags via the GlobalTags aspect.

This also follows the existing pattern freeform tags used to apply themselves to entities, so we are following the tradition of existing tags in this PR.

Reply:
I think this should be okay?
I was thinking that globalTags are just like ownership ... and since ownership can be attached to anything .. tags should be able to attach to anything.

Reply:
I think maybe woody's point is you _have to_ manually attach to everything. I don't think there's a way around this though. Unless you want the aspect, but to not return it on the entity.

Reply:
I'd suggest not adding tags to everything though. We don't have ownership for everything. Lets just add tags where it makes sense? i.e. why are we tagging users?

Reply:
Maybe users can be tagged with "sql-ninja" :)

Reply:
Correct. I don't think it's ideal to include tag properties to every possible entity and object.  I think the action massaging entity with tags can happen at GraphQL layer on demand. Of course, we need a tag API which returns tags of input urn.

Reply:
I think this is okay as written. 
globalTags is just holding an array of TagAssociations which are just the tag-urns attached to this entity. 
The real tag and its metadata are behind the Tag entity and GraphQL is being used to "decorate / massage" it in.